cluster/ci/jobs/image-mirror.yaml

```diff
@@ -281,3 +281,50 @@ items:
               - mirror
               - -f=/etc/imagemirror/knative/knative_v0_5_quay
               - --skip-multiple-scopes # quay chokes when requesting multiple
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: "image-mirror-tekton-v0.3-quay"
+    labels:
+      app: "image-mirror-tekton-v0.3-quay"
+  spec:
+    schedule: "0 * * * *"
+    concurrencyPolicy: Forbid
+    successfulJobsHistoryLimit: 1
+    failedJobsHistoryLimit: 2
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            name: "image-mirror-tekton-v0.3-quay"
+            labels:
+              app: "image-mirror-tekton-v0.3-quay"
+          spec:
+            restartPolicy: OnFailure
+            volumes:
+              - name: push
+                secret:
+                  secretName: registry-push-credentials-quay.io-openshift-pipeline
```

Comment:
Is this secret present ? (I think I didn't yet created a robot account on quay :sweat_smile:)

Reply:
I will look into it